core/field.js

```diff
@@ -537,6 +538,11 @@ Blockly.Field.prototype.getValidator = function() {
  *  functions instead.
  */
 Blockly.Field.prototype.classValidator = function(text) {
+  Blockly.utils.deprecation.warn(
+      'Field.prototype.classValidator',
+      'May 2019',
+      'December 2020',
+      'doClassValidation');
```

Comment:
Two things, doClassValidation has an underscore `doClassValidation`, and should we be consistent and recommend `Blockly.Field.prototype.doClassValidation_`